MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvxAppCompatViewPresenter.cs

```diff
@@ -487,6 +487,20 @@ protected virtual bool CloseViewPagerFragment(IMvxViewModel viewModel, MvxViewPa
             return false;
         }
 
+        protected override bool CloseFragments()
+        {
+            try
+            {
+                CurrentFragmentManager.PopBackStackImmediate();
+            }
+            catch (Exception ex)
+            {
+                return base.CloseFragments();
+                MvxTrace.Trace("Cannot close any fragments", ex);
```

Comment:
MvxTrace is obsolete now, should we use the new logging system here?

Reply:
It is used all over the code base. I think we need to see how we can replace it all.